tensorflow_probability/python/bijectors/real_nvp_test.py

```diff
@@ -97,6 +97,45 @@ def testBatchedBijectorWithMLPTransform(self):
       self.assertAllClose(x_, inverse_y_, rtol=1e-1, atol=0.)
       self.assertAllClose(ildj_, -fldj_, rtol=1e-6, atol=0.)
 
+  def testBijectorConditionKwargs(self):
+    N = 3
+    x_ = np.arange(N * 4 * 2).astype(np.float32).reshape(N, 4 * 2)
+    conditions = {
+        'a': tf.reshape(tf.range(N * 4, dtype=tf.float32), (N, 4)),
```

Comment:
Forgot about this one. Can you make this double quotations instead of single quotes (to be consistent with the rest of the file).